crates/backoff/src/lib.rs

```diff
@@ -41,7 +41,7 @@ impl Backoff {
     #[inline]
     pub fn jitter(mut self, jitter: f32) -> Self {
         assert!(
-            jitter > 0f32 && jitter < 1f32,
+            jitter >= 0f32 && jitter <= 1f32,
```

Comment:
I think it is the comment that is incorrect. Allowing a jitter of `0` would cause a panic when trying to because the range used in `thread_rng().gen_range(0..(jitter_factor * 2));` would be empty. I can't see a reason why 1 shouldn't be allowed but I am inclined to just leave the code for the two and just fix the comment